lib/bushslicer.rb

```diff
@@ -16,7 +16,7 @@ module BushSlicer
   autoload :OpenStack, "launchers/openstack"
   autoload :VSphere, "launchers/v_sphere"
   autoload :Packet, "launchers/packet"
-  autoload :OCM, "launchers/ocm"
+  autoload :OCMCluster, "launchers/o_c_m_cluster"
```

Comment:
Hi @akostadinov is there any reason for using `o_c_m_cluster` instead of `ocm_cluster`?

Reply:
@apodhrad , it can work both ways but this is the convention from converting snake case to camel case in the projects, that's why I prefer to use it.